src/sentry/relay/config/__init__.py

```diff
@@ -256,7 +265,7 @@ def get_metrics_config(timeout: TimeChecker, project: Project) -> Mapping[str, A
                 "granularitySeconds": quota["granularity_seconds"],
             },
             "limit": quota["limit"],
-            "scope": "organization",
+            "scope": options.get("scope", "organization"),
```

Comment:
You query a global `scope` option for this?

Reply:
Nice catch, thank you

Reply:
Whoops